### DISCUSSION TITLE
试以政策文件简析党政关系 | /project/pgr/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#624] [Announcements]
这是一次失败的尝试，整个过程中时常感到题目太大而力所不能及，虽然所得所获甚少，但明白了一个无用的结论——用政府公开的数据不足以分析出党政关系的全貌。之所以仍然要坚持写完这篇文章，这是因为，是因为，因为……为了耕点什么。

https://yuanfan.rbind.io/project/pgr/

**Comment by rexwangcc** [comment]
Interesting reading!

**Reply by earfanfan** [reply]
哈，3Q!
